Ports/aclock/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=aclock
+version=2.3
+# the github repo contains way too many different ports for other platforms and binaries, only fetch the file we need
+files="https://github.com/tenox7/aclock/raw/master/sources/aclock-unix-curses.c aclock-${version}.c 2b098996409c4740f492fb8fd5a63cb5e3e15283c2f7e3f06c75d6a9ad916669"
```

Comment:
Please pin this to a specific revision (tag, etc.), otherwise the download will break once it's updated the next time.